src/time_integration/paired_explicit_runge_kutta/methods_PERK3.jl

```diff
@@ -0,0 +1,410 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+using DelimitedFiles: readdlm
+
+@muladd begin
+#! format: noindent
+
+# Initialize Butcher array abscissae c for PairedExplicitRK3 based on SSPRK33 base method
+function compute_c_coeffs(num_stages, cS2)
+    c = zeros(num_stages)
+
+    # Last timesteps as for SSPRK33, see motivation in
+    # https://doi.org/10.48550/arXiv.2403.05144
+    c[num_stages - 1] = 1
+    c[num_stages] = 0.5
+
+    # Linear increasing timestep for remainder
+    for i in 2:(num_stages - 2)
+        c[i] = cS2 * (i - 1) / (num_stages - 3)
+    end
+
+    return c
+end
+
+# Compute residuals for nonlinear equations to match a stability polynomial with given coefficients,
+# in order to find A-matrix in the Butcher-Tableau
+function PairedExplicitRK3_butcher_tableau_objective_function(a_unknown, num_stages,
```

Comment:
I was wondering if this should not live in `TRixiNLsolveExt.jl`, as we only used it there, right?

Reply:
True!